### PR TITLE
[1.1.x] RHIDP-1583: Add details for favicon customization

### PR DESCRIPTION
**Pull request by abrennan89**
**Versions:**
1.1

**Jira link:**
- https://issues.redhat.com/browse/RHIDP-1583

**Preview link:**
- https://redhat-developer.github.io/red-hat-developers-documentation-rhdh/pr-281/getting-started-rhdh/#ref-additional-rhdh-customizations_rhdh-getting-started

**Reviews:**
- [ ] SME: @invincibleJai 
- [ ] QE: TBA
- [ ] Peer review: TBA

**Notes**
This PR implements the 1.1.x changes from the following PRs:
- https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/260
- https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/280

### CODE REVIEW

File: getting-started/ref-additional-rhdh-customizations.adoc [hunk 1]
```diff
@@ -67,4 +68,20 @@ In the previous configuration,
 ** `hsl()`
 ** `hsla()`
 ** `color()`
---
\ No newline at end of file
+
+[id="ref-additional-rhdh-customizations-favicon"]
+=== Customizing the favicon
+
+The favicon uses the same Base64 encoded image as the icon logo specified in the `app.branding.iconLogo` field of the `app-config.yaml` file.
```
Comment (invincibleJai) [inline getting-started/ref-additional-rhdh-customizations.adoc:75]:
```suggestion
The favicon uses the same Base64 encoded image as the icon logo specified in the `app.branding.iconLogo` field of the `app-config-rhdh.yaml` file.
```

should we have as above in this file `app-config-rhdh.yaml` was used?

**Comment by invincibleJai** [comment]
@Zaperex I tried to verify this but couldn't, can you pls check if this works as expected?

**Comment by invincibleJai** [comment]
Favicon customization is not working as expected, we have a ticket for this here https://github.com/janus-idp/backstage-showcase/issues/777

So I don't think we should be capturing this as part of docs and can add it in known issues may be.

**Comment by abrennan89** [comment]
Closing PR since this functionality currently isn't working

**Pull request closed without merging**